tls/s2n_tls13.h

```diff
@@ -26,6 +26,7 @@ int s2n_enable_tls13();
 int s2n_disable_tls13();
 bool s2n_is_valid_tls13_cipher(const uint8_t version[2]);
 
-bool s2n_is_hello_retry_required(struct s2n_connection *conn);
 bool s2n_is_hello_retry_handshake(struct s2n_connection *conn);
+bool s2n_is_hello_retry_message(struct s2n_connection *conn);
 int s2n_set_hello_retry_required(struct s2n_connection *conn);
+int s2n_hello_retry_validate(struct s2n_connection *conn);
```

Comment:
Does this ever need to be called outside of server_hello? Can it go back to being a static method?